frontend/src/components/ProjectMemberPanel.vue

```diff
@@ -1,8 +1,27 @@
 <template>
-  <div>
-    <p class="text-lg font-medium leading-7 text-main">
+  <div class="content-center">
+    <span class="text-lg font-medium leading-7 text-main">
       {{ $t("project.settings.manage-member") }}
-    </p>
+    </span>
+
+    <span
+      :class="
+        allowSyncVCS
+          ? 'ml-4 text-sm select-none normal-link'
+          : 'ml-4 text-sm select-none cursor-not-allowed'
+      "
+      @click.prevent="
+        () => {
+          if (allowSyncVCS) {
+            syncFromVCS();
+          }
+        }
+      "
+    >
+      <heroicons-outline:refresh class="inline align-text-top" />
+      {{ $t("project.settings.sync-from-vcs") }}
+    </span>
+
     <div v-if="allowAddMember" class="mt-4 w-full flex justify-start">
```

Comment:
If syncing in VCS mode, then we should not allow user to manually add the member or edit the role below

Reply:
the current implementation may not support this feature, since we only store role providers at a project-member scale. 
It should be fine if the owner wants to apply some minor tweaks towards the role, and the current implementation would free users from tedious work.
e.g. 
1. say if the owner wants to add a new owner or wants to promote somebody from developer to owner, she would have to first go to GitLab apply such changes and manually click the SYNC button at bytebase

Reply:
And since GitLab does not have relevant webhooks for role events, It is impossible for us to maintain consistency in members' roles between GitLab and Bytebase.

Reply:
So looks like we need to add a role_provider to the project table as well.

The main selling point of this feature is user only needs to manage project role in GitLab.

If we still allow user to manually change the role, then it actually adds the extra headache. Because now there are two places determining the role.

Reply:
Then how should we deal with the members who exist in Bytebase before the owner click the 'sync' button? 

I think allowing users to change members' roles at Bytebase provides some flexibility and the current implementation is just fine.